src/analytics/src/main/scala/osmesa/analytics/oneoffs/AugmentedDiffStreamProcessor.scala

```diff
@@ -272,13 +254,7 @@ object AugmentedDiffStreamProcessor extends CommandApp(
 
             this.partitionId = partitionId
             this.version = version
-
-            connection = (databaseUser, databasePass).mapN { (usr, pass) =>
-              DriverManager.getConnection(s"jdbc:${databaseUri.toString}", usr, pass)
-            }.getOrElse {
-              DriverManager.getConnection(s"jdbc:${databaseUri.toString}")
-            }
-
+            connection = DBUtils.getJdbcConnection(databaseUri)
```

Comment:
So tempting to make this method extension on `URI` but those tend to become magical so I'm resisting the urge.

Reply:
ex: `databaseUri.getJdbcConnection()`